### ISSUE TITLE
small FR: Midi File Drag'n'Drop Support 

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#687]
Hey,

another smaller one. Not a biggie but would be convenient to have.
A lot of plugins that generate midi have the ability to drag midi out of the Plugin-GUI into the DAW.

This is very convenient. I think this could be handled the same way as audio files are handled now.
Just add some tracks (with either new sample players or just "dummy" tracks/empty ones)
for every track inside the midi file.

Also it seems like the midi-import via the file-browser is prone to crashes?  

Best,
tL.

**Comment by kmatheussen** [comment]
> Also it seems like the midi-import via the file-browser is prone to crashes? 

Yes, it seems like the player is not stopped when importing midi file. You need to stop the player first manually, at least until next release.

**Comment by Lykdraft** [comment]
Hm, forgot to answer here... Well, for my purposes the midi-import is useless, as Radium opens a new song (or clears the current one) when importing midi. That's not good. Doesnt have any use for what I do unfortunately.

Most of the time I want to **add** a Midi File to your song, not the other way around.
Think about all the pattern generators and stuff like Maschine, Geist, all midi-stepsequencers or other generators. Basically all the stuff where you can drag midi from the plug directly to the host. I also get often midi-parts send for specific intstruments for remix purposes and so on and so forth.

So we have actually a complete different issue here. Drag'n'Drop of course will not help when the song gets cleared.. I would rather prefer that the midi-import respects the openend song than having drag'n'drop support that clears the song... ;) What are your thoughts on that?

Best,
tL.

**Comment by kmatheussen** [comment]
Okay, it's easy to remove the code that deletes the already existing blocks when importing midi.

**Comment by Lykdraft** [comment]
Awesome. That's good news!
